code-challenges.js

```diff
@@ -59,3 +161,14 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+function diff (num1, num2) {
+    if( num1 > num2) {
+        return num1 - num2
+    } else
+        return num2 - num1
+}
+
+console.log(diff(number1, number2)) 
+console.log(diff(number3, number4))
```

Comment:
This looks great!  I know these problems are tough, but your hard work is paying off!